similaritymeasures/similaritymeasures.py

```diff
@@ -879,3 +879,111 @@ def dtw_path(d):
     path = np.array(path)
     # reverse the order of path, such that it starts with [0, 0]
     return path[::-1]
+
+def mae(exp_data, num_data, p=2, threshold=1000000):
```

Comment:
Sorry for my mistake. Distance matrix did not do what I want.

Remove `p` and `threshold` as options, they won't make sense in the final version.